src/client/components/CheckoutDetails/PageSection.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+
+const { gray200 } = colorsV3
+
+export const PageSection = styled.div`
+  background-color: ${gray200};
+  width: 100%;
+  max-width: 628px;
```

Comment:
Maybe use relative units rather them `px`?

Reply:
Hmm, should we use that for `width` and `height` values as well? 🤔 Maybe we should, I'm trying to think what happens with the layout if the browser font size changes. But yes, maybe it's good that everything scales in the same way.